100of100Service/src/main/java/dev/coms4156/project/MysqlConnection.java

```diff
@@ -53,11 +53,16 @@ public Employee getEmployee(int organizationId, int externalEmployeeId) {
       pstmt.setInt(2, internalEmployeeId);
       ResultSet rs = pstmt.executeQuery();
       if (rs.next()) {
-        return new Employee(
```

Comment:
Your code has nothing wrong, but you should be able to use the full constructor of Employee to set everything at once.